src/_help/organizations/personal-user-account-migration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Personal user account migration
+---
+
+- TOC
+{:toc}
+
+Since the launch of Bump.sh, we've noticed that the approach we initially chose was no longer suited to our users. Indeed, they heavily utilize Bump.sh collaboratively.
+
+To accommodate this evolution and prepare for future features, we've decided to transform personal user accounts into organizations.
+
+## Why this change?
+
+Initially, Bump.sh was first designed around individual usage before gradually shifting towards team collaboration. However, it's this latter need that has become widely predominant.
+
+We observed that personal user accounts sometimes caused confusion, especially for users signing up to join an organization. Conversely, some personal projects could grow enough to transition to team collaboration.
+
+## What does this change mean for me?
+
+For our new users, the change mainly occurs during registration, where you'll be asked to create your organization. You'll enjoy all the features of organizations during your free trial. At the end of it, you can choose to continue benefiting from the full potential of organizations or switch to a free plan.
```

Comment:
I don't think this is useful: we should probably focus on explaining the changes to our impacted users.